### PR TITLE
New blog post NgOptimizedImage

### PR DESCRIPTION
**Pull request by kaslee**
Blog post for Angular NgOptimizedImage

Changes proposed in this pull request:

- New blog post - one new file added

### CODE REVIEW

**Comment by kaslee** [comment]
Source document was reviewed by @rachelandrew

File: site/en/blog/angular_ngoptimizedimage/index.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+The downside to the `srcset` attribute is that it can be cumbersome to implement. Manually writing out `srcset` values means adding multiple lines of markup to each image element in your app, complete with multiple custom URLs for each `srcset`. You also have to decide on a set of breakpoints, which is complicated, as they can represent both screen densities and the viewport sizes of common devices.
+
+That’s why adding **[automated srcset generation](https://angular.io/guide/image-directive#request-images-at-the-correct-size-with-automatic-srcset)** into the NgOptimizedImage directive was a major post-launch milestone. With this addition, any application using a CDN that supports image resizing can get full, customizable, srcsets automatically added to every image generated with the NgOptimizedImage directive. 
+
+We’ve included a simplified API for setting the `sizes` property, which is used to ensure that each image gets the correct _type _of `srcset`. If you don’t include a `sizes` attribute, we know that the image is meant to be fixed-size, and should get a density-dependent srcset, like the following:
```
Comment (rachelandrew) [inline site/en/blog/angular_ngoptimizedimage/index.md:45]:
```suggestion
We’ve included a simplified API for setting the `sizes` property, which is used to ensure that each image gets the correct type of `srcset`. If you don’t include a `sizes` attribute, we know that the image is meant to be fixed-size, and should get a density-dependent srcset, like the following:
```

File: site/en/blog/angular_ngoptimizedimage/index.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+This kind of srcset ensures that images are served at a size that takes the user’s device pixel density into account. 
+
+On the other hand, if you do include the `sizes` property, `NgOptimizedImage` generates a responsive srcset that includes breakpoints for many common device and image sizes, using this default list of breakpoints:
+
+
+### Preconnect generation
+
+To improve LCP, it’s important to reduce the time your users spend downloading the LCP image. In the previous section, you saw how `srcset` can assist  by transferring smaller image files, but an equally important optimization is to start the transfer as soon as possible. One way to do that is by using `link rel=”preconnect”` tags to jump-start the connection to your image domain.
```
Comment (rachelandrew) [inline site/en/blog/angular_ngoptimizedimage/index.md:54]:
```suggestion
To improve LCP, it’s important to reduce the time your users spend downloading the LCP image. In the previous section, you saw how `srcset` can assist  by transferring smaller image files, but an equally important optimization is to start the transfer as soon as possible. One way to do that is by using `link rel="preconnect"` tags to jump-start the connection to your image domain.
```

**Review by rachelandrew — APPROVED** [review]

**Comment by atcastle** [comment]
@kaslee  I don't see the code snippets in this PR. Does that need to be added here or is it handled separately? CC: @kara

**Comment by kaslee** [comment]
@atcastle - I had missed the code snippets and have included them now. Please check.

**Comment by atcastle** [comment]
LGTM

**Comment by kara** [comment]
@rachelandrew This should be good to merge I think? (We don't have merge access ourselves)

**Pull request merged**